llvm/test/CodeGen/NVPTX/f16-instructions.ll

```diff
@@ -143,11 +143,21 @@ define half @test_fsub(half %a, half %b) #0 {
 ; CHECK-NOF16-NEXT: cvt.rn.f16.f32 [[R:%rs[0-9]+]], [[R32]]
 ; CHECK-NEXT: st.param.b16    [func_retval0+0], [[R]];
 ; CHECK-NEXT: ret;
-define half @test_fneg(half %a) #0 {
+define half @test_old_fneg(half %a) #0 {
   %r = fsub half 0.0, %a
   ret half %r
 }
 
+; CHECK-LABEL: test_fneg(
+; CHECK-DAG:  ld.param.b16    [[A:%rs[0-9]+]], [test_fneg_param_0];
```

Comment:
Nit, what does a single CHECK-DAG mean?  I thought this was only meaningful if you had two or more of them.  https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-dag-directive